components/card/src/vwc-card.scss

```diff
@@ -78,6 +91,10 @@ header {
 	height: 20px;
 }
 
+::slotted([slot="top-action"i]) {
+	margin-left: auto;
```

Comment:
if the wrapper already defines itself with `space-between`, is this still needed?

Reply:
this is for the slot that's outside of the header-wrapper - but I've added to header space-between and delete it